### ISSUE TITLE
\Description from acmart is shown in HTML version

### ISSUE THREAD

**Issue by RexYuan** [arXiv/html_feedback#3784]
### Description

 \Description inside figure from acmart is shown in HTML verison

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

c94e02cd-475b-46e9-8a9a-6e59246af00d

### Paper URL

https://arxiv.org/html/2504.21634v1

### Browser

Chrome/135.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from https://github.com/brucemiller/LaTeXML/pull/2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
